package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "react-remote-resource",
-  "version": "0.2.2",
+  "version": "0.2.3",
```

Comment:
I don't know what versioning system being used! If something similar to `simver` this likely would be a breaking change. If not `simver`, at least a minor number would be good to show some significant changes.

Reply:
Yeah, this was carryover from a previous version. I was originally planning on changing the version after merging, but I guess that wouldn't make sense. I'll go ahead and change it.